packages/peregrine/lib/store/reducers/app.js

```diff
@@ -46,6 +47,18 @@ const reducerMap = {
             isOnline: false,
             hasBeenOffline: true
         };
+    },
+    [actions.htmlUpdateAvailable]: state => {
+        return {
+            ...state,
+            htmlUpdateAvailable: true
```

Comment:
Would it be sufficient to just use local state within the `App` component/talon? I think you could get away with it rather than requiring redux state.

Reply:
Using local state as you mentioned. Good suggestion 👍